Modules/SubModule_Import/xASL_imp_Deface.m

```diff
@@ -17,22 +17,20 @@ function xASL_imp_Anonymize(imPar)
 % 3. Process all anatomical files (`xASL_spm_deface`)
 %
 % -----------------------------------------------------------------------------------------------------------------------------------------------------
-% EXAMPLE:     xASL_imp_Anonymize(imPar);
+% EXAMPLE:     xASL_imp_Deface(imPar);
 % __________________________________
 % Copyright 2015-2021 ExploreASL
 
-    %% Anonymize
-    
-    % 1. Iterate over list of subjects
+    %% 1. Iterate over list of subjects
     listSubjects = xASL_adm_GetFileList(imPar.TempRoot,[],false,[],true);
```

Comment:
Not important for now as I guess nobody uses this function (yet), but this doesn't give any result, it looks for folders without a name in the `TempRoot` folder :) Or does the second input argument default to something (it shouldn't I believe).

Reply:
There really seems to be a bug related to all the renaming and stuff. Another issue here is that we have no nice way to test if the defacing happened or not, yet.

Reply:
Fix of the first problem: 490c59c

Reply:
Fix of the bug: e94cab3

Reply:
Weird. It should of course work. But maybe something happened with the directories during all the revamping. This is not a function we were testing. I will fix this.

Reply:
Seems that Michael already did that :D

Reply:
Tested and all works. Good work Michael.